.github/workflows/reusable-pip-compile.yml

```diff
@@ -29,6 +29,9 @@ name: "Refresh pinned dependencies"
       reset-branch:
         type: boolean
         default: false
+      labels:
+        type: string
+        default: ""
```

Comment:
Other places have `required: false` instead of `default: ""`. I suggest changing this for consistency.